src/pages/index.js

```diff
@@ -33,40 +60,95 @@ const cardFormValidator = new FormValidator(
   constants.cardFormElement
 );
 
-editFormValidator.enableValidation();
-cardFormValidator.enableValidation();
-
-const cardCreator = new Section(
-  {
-    items: constants.initialCards,
-    renderer: createCard,
-  },
-  ".cards__list"
+const avatarFormValidator = new FormValidator(
+  constants.validationSettings,
+  constants.avatarFormElement
 );
 
-cardCreator.renderItems();
+editFormValidator.enableValidation();
+cardFormValidator.enableValidation();
+avatarFormValidator.enableValidation();
 
 const userProfileInfo = new UserInfo({
   profileName: ".profile__title",
   profileJob: ".profile__subtitle",
+  profileAvatar: ".profile__image",
 });
 
+api
+  .getUserInfo()
+  .then((data) => {
+    console.log(">>USER DATA:", data);
+    userProfileInfo.setUserInfo({
+      profileName: data.name,
+      profileJob: data.about,
+      profileAvatar: data.avatar,
+    });
+  })
+  .catch((err) => console.error(err));
+
 function handleProfileSubmit(formValues) {
-  const profileName = formValues.title;
-  const profileJob = formValues.subtitle;
-  userProfileInfo.setUserInfo({ profileName, profileJob });
-  profilePopup.close();
+  profilePopup.setLoadingState(true);
+
+  api
+    .updateUserInfo({ name: formValues.title, about: formValues.subtitle })
+    .then((updatedData) => {
+      userProfileInfo.setUserInfo({
+        profileName: updatedData.name,
+        profileJob: updatedData.about,
+      });
```

Comment:
* When I edit the profile data, the avatar disappears https://snipboard.io/OpHURX.jpg


```js
    .then((updatedData) => {
      userProfileInfo.setUserInfo({
        profileName: updatedData.name,
        profileJob: updatedData.about,
        profileAvatar: updatedData.avatar  // set the avatar here
      });
```